app/src/sharedTest/java/org/oppia/app/profile/ProfileChooserFragmentTest.kt

```diff
@@ -48,15 +49,18 @@ import javax.inject.Singleton
 @RunWith(AndroidJUnit4::class)
 class ProfileChooserFragmentTest {
 
-  @Inject lateinit var profileTestHelper: ProfileTestHelper
-  @Inject lateinit var context: Context
+  @Inject
+  lateinit var profileTestHelper: ProfileTestHelper
+  @Inject
+  lateinit var profileManagementController: ProfileManagementController
+  @Inject
+  lateinit var context: Context
```

Comment:
These should be in single line, example:
` @Inject lateinit var context: Context`